app/javascript/consent-form.vue

```diff
@@ -307,7 +307,10 @@ export default {
       if (!await this.showSpinner(this.saveAnswers)) {
         return;
       }
-      if ( this.modalFallback && this.checkboxAgreement.answer == 'no') {
+
+      const doesAgreeToAll = this.checkboxAgreementQuestions.every(
```

Comment:
Lol! This code looked so clear to me... I got distracted and it took me moment to remember I was looking at JS and not Ruby 😂